db/system_distributed_keyspace.cc

```diff
@@ -96,8 +97,27 @@ schema_ptr cdc_desc() {
         return schema_builder(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_DESC, {id})
                 /* The timestamp of this CDC generation. */
                 .with_column("time", timestamp_type, column_kind::partition_key)
-                /* The set of stream identifiers used in this CDC generation. */
+                /* For convenience, the list of stream IDs in this generation is split into token ranges
```

Comment:
s/token ranges/vnodes?

Reply:
Dunno, these ranges are vnodes when they are created - true, but after a topology change - not necessary (for old generations)

Reply:
Fair enough - it is true for old generations.